src/services/validatorSchema.ts

```diff
@@ -0,0 +1,17 @@
+import * as yup from 'yup';
+
+export const inventoryItemSchema = yup.object({
+    user_id: yup.string().defined("All fields must be completed"),
```

Comment:
I like that you extracted each schema into export in a separate file for reuse 👍 
Should we have a separate file for the schemas or would it make sense to collocate the schemas with the related services?

Reply:
another thing, each error should be specific. If the user_id field is missing I shoulnd't have a "all fields must be completed" error since I wouldn't know which field I'm missing. Something like "user_id field must be provided" would be more explicit 👍